src/connection-manager/connections.ts

```diff
@@ -55,11 +62,31 @@ async function handleWsMessageTypes(
     if (ledger_hashes.includes(validationData.ledger_hash)) {
       validationData.networks = networks
     }
-    void agreement.handleValidation(data as ValidationRaw)
+    const validationNetworkDb: DatabaseValidator | undefined = await query(
+      'validators',
+    )
+      .select('*')
+      .where('signing_key', validationData.validation_public_key)
+      .first()
+    const validationNetwork =
+      validationData.networks ?? validationNetworkDb?.networks
+    if (validationNetwork) {
+      validationData.ledger_fee = network_fee.get(validationNetwork)
+    }
```

Comment:
So is this setting the validator's default fee vote to the current network state?

Reply:
Won't this override a fee vote by the validator? Shouldn't it only set this if `validationData.ledger_fee` isn't set?

Reply:
the `ledger_fee` field is a field that I added (not included in the validation response from the stream) containing `base_fee`, `reserve_base`, `reserve_inc`. If we don't see one of these 3 fields in the validation (so no new fee is voted), then we will use `ledger_fee` to fill in the data.